index.d.ts

```diff
@@ -243,13 +243,15 @@ declare module 'warframe-items' {
         pellet?: Pellet;
         duration?: number;
         speed?: number;
-        falloff?: {
+        falloff?: Falloff;
+    }
+    
+    interface Falloff {
             start: number;
             end: number;
             reduction: number;
-        };
     }
-
+    
```

Comment:
drop the trailing spaces?

Reply:
Oh right, definetly
(now if I resolve this will it do it for me...)

Reply:
nah, i'd have to add a suggestion

Reply:
ah, looks like you force pushed 👍